src/SophiApp/Localizations/IT.xaml

```diff
@@ -3,26 +3,26 @@
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
     xmlns:system="clr-namespace:System;assembly=mscorlib">
 
-    <system:String x:Key="Conditions.IsSingleSession.Login">Per poter eseguire SophiApp, devi possedere un account con i permessi di amministratore.</system:String>
+    <system:String x:Key="Conditions.IsSingleSession.Login">Per poter eseguire SophiApp, devi avere un account con i permessi di amministratore.</system:String>
     <system:String x:Key="Conditions.NewVersion.HasNew">é disponibile una nuova versione di SophiApp.</system:String>
-    <system:String x:Key="Conditions.NewVersion.OpenGitHub">Apri la repository su GitHub</system:String>
-    <system:String x:Key="Conditions.NoRebootRequired.RebootPending">Il Pc sta per essere riavviato.</system:String>
-    <system:String x:Key="Conditions.NoRebootRequired.PleaseReboot">Si prega di riavviare il PC prima di eseguire la SophiApp.</system:String>
-    <system:String x:Key="Conditions.OneInstanceOnly">Non eseguire più istanze di SophiApp contemporaneamente.</system:String>
-    <system:String x:Key="Conditions.OsBuildVersion.NotCompatible">SophiApp non è compatibile con la tua versione di Windows.</system:String>
+    <system:String x:Key="Conditions.NewVersion.OpenGitHub">Apri repository GitHub</system:String>
+    <system:String x:Key="Conditions.NoRebootRequired.RebootPending">Il computer sta per essere riavviato.</system:String>
+    <system:String x:Key="Conditions.NoRebootRequired.PleaseReboot">Prima di eseguire la SophiApp riavvia il computer.</system:String>
+    <system:String x:Key="Conditions.OneInstanceOnly">Non eseguire contemporaneamente più istanze di SophiApp.</system:String>
+    <system:String x:Key="Conditions.OsBuildVersion.NotCompatible">SophiApp non è compatibile con questa versione di Windows.</system:String>
```

Comment:
"l'attuale" maybe sounds better?

Reply:
For me "l'attuale" seems more generic then "questa" that it's specfic.

Reply:
L'attuale si riferisce alla versione che ora è installata, che poi potrà venir aggiornata....
(scrivo in italiano per capirci meglio)

Reply:
ok for "l'attuale".